client/src/partials/fiscal/templates/modals/fiscal.closing.modal.html

```diff
@@ -0,0 +1,111 @@
+<form name="ClosingFiscalYear" bh-submit="$ctrl.stepForward(ClosingFiscalYear)" bh-form-defaults novalidate>
+  <div class="modal-header bg-danger text-danger">
+    <i class="fa fa-lock"></i>
+    <strong>{{ 'FISCAL.CLOSING_FISCAL_YEAR' | translate }}</strong>
+  </div>
+
+  <div class="modal-body">
+    <div ng-show="!$ctrl.fiscal.locked">
+
+      <h1 class="text-center">
+        {{'FORM.INFO.CLOSE_FISCAL_YEAR' | translate }}<br>
+        <strong>{{ $ctrl.fiscal.label }}</strong>
+      </h1>
+
+      <div ng-switch="$ctrl.steps">
+
+        <div ng-switch-default>
+
+          <!-- first step  -->
+          <dl class="dl-horizontal">
+            <!-- incomes  -->
+            <h3 class="bg-success text-success">
+              <dt>{{ 'FORM.LABELS.PROFIT' | translate }}</dt>
+              <dd>
+                <div class="text-right">
+                  {{ $ctrl.profit | currency }}
+                </div>
+              </dd>
+            </h3>
+
+            <!-- expenses  -->
+            <h3 style="margin: 10px 0px;" class="bg-danger text-danger">
+              <dt>{{ 'FORM.LABELS.CHARGE' | translate }}</dt>
+              <dd>
+                <div class="text-right">
+                  {{ $ctrl.charge | currency }}
+                </div>
+              </dd>
+            </h3>
+
+            <!-- results  -->
+            <h3 style="border-top: 1px dashed #ccc; margin-top: 10px;" class="text-bold">
+              <dt>{{ 'FORM.LABELS.RESULT' | translate }}</dt>
+              <dd>
+                <div class="text-right" ng-class="{'text-danger' : $ctrl.globalResult < 0, 'text-success' : $ctrl.globalResult > 0 }">
+                  {{ $ctrl.globalResult | currency }}
```

Comment:
When you specify currency, it is best practice to specify the `currency_id` as well.

This should probably be
```html
{{ $ctrl.globalResult | currency:currency_id }}
```

The same should be the case for all your other `currency` fields on this page.